proof/proof_test.go

```diff
@@ -1467,7 +1467,10 @@ func TestProofUnknownOddType(t *testing.T) {
 
 func init() {
 	logWriter := build.NewRotatingLogWriter()
-	logger := logWriter.GenSubLogger(Subsystem, func() {})
-	logWriter.RegisterSubLogger(Subsystem, logger)
+	logConfig := build.DefaultLogConfig()
+	logMgr := build.NewSubLoggerManager(
+		build.NewDefaultLogHandlers(logConfig, logWriter)...,
+	)
+	logger := logMgr.GenSubLogger(Subsystem, func() {})
 	UseLogger(logger)
```

Comment:
I dont think we actually want to write to a file here right? ie, we dont need to rotating log writer anymore - we used to cause we needed that just to get the stdout logger to work but now these have been separated. So I think we can just:

```
func init() {
	logger := btclog.NewSLogger(btclog.NewDefaultHandler(os.Stdout))
	UseLogger(logger.SubSystem(Subsystem))
}

```

Reply:
Ah yes, thanks! Updated.